src/visualization/classification/extra_trees_vis.py

```diff
@@ -0,0 +1,147 @@
+        qgv.setStyleSheet("background-color:white;")
+        qgv.new(Dot(Graph("graph", graph_type=GraphType.DirectedGraph), font=QFont("Arial", 12), margins=[10, 500]))
+        nodes = {}
+        for line in lines:
+            if len(line) == 0:
+                continue
+            if line.startswith('node') or line.startswith('edge'):
+                continue
+            param = None
+            if '[' in line:
+                value, param = line.split('[', 1)
+                param = param.rsplit(']', 1)[0]
+            else:
+                value = line.rsplit(';', 1)[0]
+            value_list = value.split()
+            if len(value_list) == 1:
+                node = value_list[0]
+                param_dict = self.dict_of_param(param)
+                if 'label' in param_dict.keys():
+                    label_lines = param_dict["label"].split('\n') if '\n' in param_dict["label"] else [param_dict["label"]]
+                    max_len = max(label_lines, key=len)
+                    rect = qgv.engine.fm.boundingRect(max_len)
+                    width = rect.width() + 20
+                    height = rect.height() * len(label_lines) + 20
+                    param_dict['size'] = (width, height)
+                param_dict['pos'] = [float(value) for value in graph_info.get_node(node).attr['pos'].split(',')]
+                param_dict['pos'][0] = int(param_dict['pos'][0])
+                param_dict['pos'][1] = int(pos_delta - param_dict['pos'][1])
+                nodes[node] = qgv.addNode(qgv.engine.graph, str(node), **param_dict)
+            elif len(value_list) == 3:
+                node1 = value_list[0]
+                node2 = value_list[2]
+                param_dict = self.dict_of_param(param)
+                qgv.addEdge(nodes[node1], nodes[node2], param_dict)
+            else:
+                raise ValueError("Invalid format of dot string")
+        qgv.build()
+        return qgv
+
+    def click_listener(self, button_type: str):
+        match button_type:
+            case 'next':
+                new_graph = min(len(self.graphs), self.current_graph + 1)
+                if new_graph == self.current_graph:
+                    return
+                self.current_graph = new_graph
+                self.update_graph()
+            case 'prev':
+                new_graph = max(1, self.current_graph - 1)
+                if new_graph == self.current_graph:
+                    return
+                self.current_graph = new_graph
+                self.update_graph()
+            case 'random':
+                self.current_graph = randint(1, len(self.graphs))
+                self.update_graph()
+
+    def update_graph(self):
+        for i in reversed(range(self.graph_group_layout.count())):
+            self.graph_group_layout.itemAt(i).widget().setParent(None)
+        self.graph_group_layout.addWidget(self.graphs[self.current_graph - 1])
+        self.num_label.setText(f"Actual graph: {self.current_graph}")
```

Comment:
```suggestion
        self.num_label.setText(f"Current graph: {self.current_graph}")
```
or alternatively `f"Tree #{self.current_graph}"`